packages/libs/eda/src/lib/map/analysis/hooks/standaloneVizPlugins.ts

```diff
@@ -101,7 +101,7 @@ export function useStandaloneVizPlugins({
 
     const pluginBasics = {
       configurationComponent: ZeroConfigWithButton,
-      isConfigurationValid: t.undefined.is,
+      isConfigurationComplete: t.undefined.is,
```

Comment:
how does this relate to our decision to not have undefined as default compute config (in favor of `{}`) in each of the apps?

Reply:
is a standalone viz one without a compute? not sure im following..

Reply:
Yes, these standalone vizs use a "compute" without config